app/models/group.rb

```diff
@@ -13,11 +13,11 @@ def resources
   end
 
   def add_admin(user)
-    groups_users.create(user: user, admin: true)
+    groups_users.new(user: user, admin: true).save
```

Comment:
Alright, I'm going to be completely honest - I don't remember. I had a reason, maybe even a good one, but I'm not sure why this change was needed. I'm going to look into it and remove it if it's not needed. Else, I'll let you know why I changed it. Deal? ;)